### PR TITLE
virtio_fs: enable virtiofs debug log in windows vm

### PR DESCRIPTION
**Pull request by xiagao**
Need to config virtiofs debug log in register instead.
ID: 2056741
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by xiagao** [comment]
@YongxueHong Could you help to check?Thanks.

**Comment by leidwang** [comment]
@YongxueHong Could you please help to review this patch?Thanks a lot!

File: qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg [hunk 1]
```diff
@@ -47,11 +47,15 @@
...
         install_cmd = 'msiexec /i WIN_UTILS:\winfsp-1.7.20172.msi /qn'
         check_installed_cmd = 'dir "%s" |findstr /I winfsp'
-        viofs_log_file = C:\\viofs_log.txt
-        viofs_sc_create_cmd = 'sc create VirtioFsSvc binpath="%s -d -1 -D ${viofs_log_file}" start=auto'
+        viofs_log_file = C:\viofs_log.txt
+        viofs_sc_create_cmd = 'sc create VirtioFsSvc binpath="%s" start=auto'
         viofs_sc_create_cmd += ' depend="WinFsp.Launcher/VirtioFsDrv" DisplayName="Virtio FS Service"'
         viofs_sc_start_cmd = 'sc start VirtioFsSvc'
         viofs_sc_query_cmd = 'sc query VirtioFsSvc'
+        viofs_reg_set_cmd = viofs_debug_enable_cmd, viofs_log_enable_cmd
```
Comment (YongxueHong) [inline qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg:55]: Actually, `viofs_reg_set_cmd` does not includes the specific command. not like `viofs_debug_enable_cmd`, `viofs_log_enable_cmd`, and so on, it looks like a set of commands or the order of commands to be executed.
I think we should rename it in a better way, like `viofs_reg_set_cmds_order`:
`viofs_reg_set_cmds_order = "${viofs_debug_enable_cmd}, ${viofs_log_enable_cmd}"`

or we don't need it at all.
The calling place we could do it like this:
```
viofs_debug_enable_cmd = params.get("viofs_debug_enable_cmd") 
viofs_log_enable_cmd = params.get("viofs_log_enable_cmd"):
if viofs_debug_enable_cmd and viofs_log_enable_cmd:
    ...
       for cmd in (viofs_debug_enable_cmd, viofs_log_enable_cmd):
       error_context.context("Set %s " % cmd, test.log.info)
       s, o = session.cmd_status_output(cmd)
```

Comment (xiagao) [reply qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg]: Thanks, update.

**Comment by xiagao** [comment]
Run a test loop.there is no regression
 (20/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.none.with_writeback.q35: STARTED
\ (20/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.none.with_writeback.q35: PASS (65.54 s)
 (21/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data.socket_group.q35: STARTED
| (21/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data.socket_group.q35: PASS (68.52 s)
 (22/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data.socket_group.unknown_group.q35: STARTED
| (22/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data.socket_group.unknown_group.q35: PASS (2.61 s)
 (23/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_symbolic_link.with_cache.auto.q35: STARTED
| (23/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_symbolic_link.with_cache.auto.q35: PASS (67.09 s)
 (24/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data_multi_backend.with_cache.auto.with_nfs_source.q35: STARTED
/ (24/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data_multi_backend.with_cache.auto.with_nfs_source.q35: PASS (76.67 s)
 (25/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data_multi_backend.with_cache.auto.xattr_on.flock_off.lock_posix_off.with_nfs_source.q35: STARTED
/ (25/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data_multi_backend.with_cache.auto.xattr_on.flock_off.lock_posix_off.with_nfs_source.q35: PASS (67.51 s)
 (26/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data_multi_backend.with_cache.auto.xattr_off.flock_off.lock_posix_off.with_nfs_source.q35: STARTED
\ (26/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_share_data_multi_backend.with_cache.auto.xattr_off.flock_off.lock_posix_off.with_nfs_source.q35: PASS (65.33 s)
 (38/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_set_capability.remove_capability.cap_dac_read_search.with_cache.auto.q35: STARTED
\ (38/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_set_capability.remove_capability.cap_dac_read_search.with_cache.auto.q35: PASS (125.15 s)
 (39/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_set_capability.remove_capability.cap_sys_admin.with_cache.none.q35: STARTED
- (39/47) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_fs_set_capability.remove_capability.cap_sys_admin.with_cache.none.q35: PASS (66.09 s)

File: qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg [hunk 2]
```diff
@@ -3,9 +3,6 @@
     only Windows
     virtio_win_media_type = iso
     cdroms += " virtio"
-    clone_master = yes
-    master_images_clone = image1
-    remove_image_image1 = yes
```
Comment (YongxueHong) [inline qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg:8]: Why do this? Does it relate to enable virtiofs debug info?
If not, we could add another commit for it in this PR.

Comment (xiagao) [reply qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg]: Soorry for my careless, it was a wrong push.
Updated.@YonxueHong

Comment (YongxueHong) [reply qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg]: > Soorry for my careless, it was a wrong push. Updated.@YonxueHong

Never mind, I see it.

**Review by YongxueHong — APPROVED** [review]

**Pull request merged**